Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -182,7 +182,7 @@ export class BotProject {
         ...mainDialog.content.$designer,
         name,
         description,
-      };
+      } as any;
```

Comment:
Why is this needed?

Reply:
the `id` field is missing in this object, triggered a ts error after we adding the `DesingerData` in sdk type. I just realized that it could be rewritten by leveraging the utility in `dialogFactory`.

Reply:
after understanding the logic here, I rewrite it like this:
![image](https://user-images.githubusercontent.com/8528761/68821691-f2cdfb00-06c9-11ea-8cc4-4addabc5087f.png)